phpcs.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
```

Comment:
**New files**

`phpcs.xml`, `phpstan.neon` and `psalm.xml` are new files for static code analysis config